src/test/resources/features/refapp-2.x/06-user/users.feature

```diff
@@ -12,20 +12,20 @@ Feature: User Account Management
     # User story: Creating user account for Data clerk
     When a user clicks add new account button
     Then the system loads add new account form
-    When a user enters data clerk details in the user account form
+    And a user enters data clerk details in the user account form
     And a user clicks on save account button
     Then the system adds user account into the users table
 
     # User story: Creating user account for Nurse
     When a user clicks add new account button
     Then the system loads add new account form
-    When a user enters nurse details in the user account form
+    And a user enters nurse details in the user account form
     And a user clicks on save account button
     Then the system adds user account into the users table
 
     # User story: Creating user account for Doctor
     When a user clicks add new account button
```

Comment:
Hey , am trying to mean, **When a user clicks add new account button** , this user story is being used several times and if we can simply reduce it, that will improve code .

Reply:
@sherrif10 according to what have observed after ensuring the `When a user clicks add new account button` the workflow fails because each user story requires that step.

So am rather going to consider each of them a separate scenarios than combining them into one. Combining them into one brings about duplicate step which they all need to use while building the test. 

If you have any opinion than the one am thinking of then would be great to hear it